datadog/conf.d/python.d/conf.yaml

```diff
@@ -3,3 +3,7 @@ logs:
    path: /busy-beaver/logs/busy_beaver_log.json
    service: busy-beaver
    source: python
+ - type: file
+   path: /busy-beaver/logs/busy_beaver_worker.json
+   service: busy-beaver
+   source: python
```

Comment:
ensure logs are being collected by datadog, reminder to rebuild before deployment